templates/document.html.slim

```diff
@@ -53,15 +55,15 @@ html lang=(attr :lang, 'en' unless attr? :nolang)
     - syntax_hl = self.syntax_highlighter
     - if syntax_hl && (syntax_hl.docinfo? :head)
       =syntax_hl.docinfo :head, self, cdn_base_url: cdn_base, linkcss: linkcss, self_closing_tag_slash: '/'
-    / If the query includes 'print-pdf', use the PDF print sheet
+
+    /! Printing and PDF exports
```

Comment:
Do we really want an HTML comment here?

Reply:
I'm exactly following reveal.js' presentation frame

Reply:
All good then 👍